docs/networking.md

```diff
@@ -22,6 +22,21 @@ routing table, and thus it requires its own subnet.  It is theoretically possibl
 with other infrastructure (but not a second cluster!), but this is not really recommended.  Certain
 `cni` networking solutions claim to address these problems.
 
+### Supported CNI Networking
+
+Two different providers are currently built into kops:
+
+1. kopeio-vxlan
```

Comment:
so ... I actually **INCLUDED** documentation in this ... cough cough ... @justinsb you can update when you have a good url for peeps to go to :)